source/languages/en/riak/dev/search/search-data-types.md

```diff
@@ -974,13 +1090,49 @@ joan_jett.batch do |jj|
   jj.registers['last_name'] = 'Jett'
   ## Joan Jett is not an enterprise customers, so we don't need to
   ## explicitly disable this flag, as all flags are disabled by default
-  jj.counters['page_visits'].increment(10)
+  jj.counters['page_visits'].increment(25)
```

Comment:
Yes, because without it, your next example is inconsistent I believe.
https://github.com/basho/basho_docs/pull/1667/files#diff-50505b42160e923067ef94162f12d6dbR1292